core/src/main/java/com/sequenceiq/cloudbreak/service/externaldatabase/DatabaseServerParameterSetter.java

```diff
@@ -0,0 +1,11 @@
+package com.sequenceiq.cloudbreak.service.externaldatabase;
+
+import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
+import com.sequenceiq.cloudbreak.service.externaldatabase.model.DatabaseServerParameter;
+import com.sequenceiq.redbeams.api.endpoint.v4.stacks.DatabaseServerV4StackRequest;
+
+public interface DatabaseServerParameterSetter {
+    void setParameters(DatabaseServerV4StackRequest request, DatabaseServerParameter serverParameter);
```

Comment:
`DatabaseServerParameter serverParameter` Where is it used?

Reply:
currently in AWS